cmd/joy/root.go

```diff
@@ -58,7 +59,7 @@ func NewRootCmd(version string) *cobra.Command {
 				if version != "" && semver.Compare(version, cfg.MinVersion) < 0 {
 					return fmt.Errorf("current version %q is less than required minimum version %q. Please update joy", version, cfg.MinVersion)
 				}
-				if !semver.IsValid(version) {
+				if !semver.IsValid(version) && os.Getenv("JOY_DEV_NO_PROMPT") != "1" {
```

Comment:
you don't need this. You can use the secret `--skip-version-check` for development builds

Reply:
I moved and renamed the env var to match that.